content/iso_changelog.md

```diff
@@ -0,0 +1,97 @@
+# Change log
+
+This document is the second edition of ISO 16739-1. The following sections describe how compatibility issues with previous release are handled in general. A full list of all changes against the previous list are shown in Chapter F.1.
```

Comment:
```suggestion
This document is the second edition of ISO 16739-1. The following sections describe how compatibility issues with previous releases are handled in general. A full list of all changes against the previous release are shown in Chapter F.1.
```

Reply:
Even better "previous edition". In ISO they call it edition, not release.